aspnetcore/tutorials/choose-web-ui.md

```diff
@@ -3,83 +3,56 @@ title: Choose an ASP.NET Core UI
 author: wadepickett
 description: Learn when to use which ASP.NET Core web UI technologies. Understand the server, client and hybrid options.
 ms.author: wpickett
-ms.date: 09/12/2023
+ms.date: 10/04/2023
 uid: tutorials/choose-web-ui
 ---
 # Choose an ASP.NET Core web UI
 
 ASP.NET Core is a complete UI framework. Choose which functionalities to combine that fit the app's web UI needs.
 
-## Benefits vs. costs of server and client rendered UI
+## ASP.NET Core Blazor
 
-There are three general approaches to building modern web UI with ASP.NET Core:
+Blazor is a full-stack web app framework, recommended for most ASP.NET web UI scenarios.
 
-* Apps that render UI from the server.
-* Apps that render UI on the client in the browser.
-* Hybrid apps that take advantage of both server and client UI rendering approaches. For example, most of the web UI is rendered on the server, and client rendered components are added as needed.
+Blazor renders content with:
 
-There are benefits and drawbacks to consider when rendering UI on the server or on the client.
+* Reusable components included in pages or whole-page, routable components.
+* Switchable rendering modes that can be chosen at runtime for server-side or client-side interactivity.
```

Comment:
This bullet seems a bit redundant with the following ones. Maybe just remove it?

```suggestion
* Switchable rendering modes that can be chosen at runtime for server-side or client-side interactivity.
```

Reply:
I will remove it.  GitHub is giving an error on this line so I will re-commit outside this one.